generated/hooks/useEventListener.hook.md

```diff
@@ -1,7 +1,8 @@
-```ts
import { useRef, useEffect, RefObject } from 'react'
+```ts
+import { useRef, useEffect, RefObject } from 'react'
 
 function useEventListener<T extends HTMLElement = HTMLDivElement>(
-  eventName: string,
+  eventName: keyof WindowEventMap,
```

Comment:
Thank you to think update this here, but don't worries, the `generated/**/*.md` is generated by script. 

You can see this in the package.json, it's named `"copy:hooks"`